### PR TITLE
Assorted cleanups

### PR DESCRIPTION
**Pull request by dnicolodi**
This has been obsoleted with the switch to Click

### CODE REVIEW

**Comment by dnicolodi** [comment]
Uhm... I shluld have forgotten a commit somewhere: It seems that beancount/projects/export_tests.py still uses some of this.

**Comment by dnicolodi** [comment]
Indeed, I forgot some commits in another branch. This grew a bit longer.

File: beancount/projects/export.py [hunk 1]
```diff
@@ -1,32 +1,14 @@
 #!/usr/bin/env python3
-"""List assets and join with attributes of account and commodities to a CSV file.
```
Comment (blais) [inline beancount/projects/export.py:2]: Oh that's a real bummer. Is there no way to provide the docstring at the top of the file?
I'd really like to keep it there. It's the first thing you see when you open the file.

**Review by blais — APPROVED** [review]
SGTM
Just the comment at the top, is it possible to do that?

File: beancount/projects/export.py (hunk 1 above)
Comment (dnicolodi) [reply beancount/projects/export.py:2]: Sure, it is possible. Fixing this I noticed that the help text gets re-wrapped wrong when output by Click.

**Comment by dnicolodi** [comment]
I moved the docstring back to the top of the source file.

**Pull request merged**